LibraBFT/ImplShared/Consensus/Types/EpochIndep.agda

```diff
@@ -49,6 +49,13 @@ module LibraBFT.ImplShared.Consensus.Types.EpochIndep where
   Instant : Set
   Instant = ℕ   -- TODO-2: should eventually be a time stamp
 
+  -- TODO-2
```

Comment:
Please put the `TODO-2`  on the same line as `postulate`, along with something like "prove it".  Also, I think this is a `TODO-1` (straightforward, requires only local changes).  See the [style guide](https://github.com/oracle/bft-consensus-agda/blob/main/STYLE.md) and [`TODO` guide](https://github.com/oracle/bft-consensus-agda/blob/main/TODO.md).